docs/api/covidcast-signals/safegraph.md

```diff
@@ -47,3 +50,39 @@ size. The same method is used for aggregation to states.
 SafeGraph provides this data with a three-day lag, meaning estimates for a
 specific day are only available three days later. It may take up to an
 additional day for SafeGraph's data to be ingested into the COVIDcast API.
+
+
+# SafeGraph Weekly Patterns
+
+* **Source name:** `safegraph`
+* **Number of data revisions since 23 June 2020:** 0
+* **Date of last change:** never
+* **Available for:** county, MSA, HRR, state (see [geography coding docs](../covidcast_geography.md))
+
+Data source based on
+[Weekly Patterns](https://docs.safegraph.com/docs/weekly-patterns) dataset. SafeGraph provides this data for
+different points of interest ([POIs](https://docs.safegraph.com/v4.0/docs#section-core-places)) considering individual census block groups, using differential privacy to protect individual people's privacy in data.
+
+Delphi gathers the number of daily visits to POIs of certain types(bars, restaurants, etc.) 
```

Comment:
If you use an acronym (POI in this case) then you should clarify what it means if it is the first time it is used on this page. A bit redundant since most know what POI is, but for the sake of those that don't, style rules ask that acronyms be worded vs shortened. 

The same could be said for the first use of MSA and HRR at the top of this section, should also be done for each page when first referenced on that given page.

Reply:
This is referenced because SafeGraph has its own list of points of interests where people can find them through the links. And this is also [how](https://docs.safegraph.com/docs/places-schema) Safegraph first introduced the idea of POI. I think this is different from MSA, HRR?

Reply:
Sounds good, thanks for clarifying. 

The MSA and HRR would be a separate comment then. Where the acronym, for clarity's sake, should be spelled out with the acronym beside it. The same way you introduce a name (i.e. Bernard and Shaw Rentals (B&SR) at the top of an article and then use only the condensed version after (B&SR). Of course, using condensed titles is dependent on stylistic preferences, but wouldn't hurt for this page and already published pages as well (which I have noted in the API review that I am conducting).

Reply:
Thanks!